api/sales_dashboard/templates/sales_dashboard/home.html

```diff
@@ -18,9 +18,13 @@ <h1 class="h2">Organisations</h1>
             </form>
           </div>
           <div class='float-left'>
-            <form method="POST" action="{% url 'sales_dashboard:update-organisation-subscription-information-caches' %}">
+            <form method="POST" action="{% url 'sales_dashboard:update-organisation-subscription-information-influx-caches' %}">
               {% csrf_token %}
-              <input type="submit" value="Update subscription information caches" class="btn btn-primary m-1">
+              <input type="submit" value="Update subscription information caches from influx" class="btn btn-primary m-1">
+            </form>
+            <form method="POST" action="{% url 'sales_dashboard:update-organisation-subscription-information-chargebee-caches' %}">
+              {% csrf_token %}
+              <input type="submit" value="Update subscription information caches from chargebee" class="btn btn-primary m-1">
```

Comment:
I guess the ideal here would be to have a separate form (perhaps in a modal) that has 2 check boxes and allows you to define which caches to update. My _slight_ concern here is that I'm not too sure what will happen if someone presses both buttons in quick succession.

Reply:
Do we really need to keep both options? I men the update Chagebee should be necessary only once right? hen we could disable it, maybe using a flag?

Reply:
Only 2 buttons were left, one that updates Influx data, and another that updates both(Influx and Chargebee).